templates/component-mapper/src/src/files/date-picker.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DatePicker = () => 'MultipleChoiceList';
```

Comment:
Wrong string. I think we can also think about using simple JSX markup using input types to show of these trivial components.